rpm/etcd3-gateway.spec

```diff
@@ -0,0 +1,45 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           etcd3-gateway
+Summary:        etcd3 gateway Python Client
+Epoch:          1
```

Comment:
Why is that?
There is no need to start with Epoch 1 for a new package

Reply:
Sorry, ignorance on my part.